controller/src/main/java/io/pravega/controller/store/stream/PersistentStreamBase.java

```diff
@@ -430,7 +430,7 @@ public String getScopeName() {
         } else {
             log.debug("scale conflict for stream {}/{} with segments to seal {}", scope, name, sealedSegments);
```

Comment:
Should the log level be higher here, like info or warn?

Reply:
not really, because with all scale operations serialized, we will only get into this situation in failover case when a request is rerun while another scale was ongoing. 
its a harmless exception that should be ignored.